Change societal amenities color

Adamant36 opened the pull request:
This PR changes the color of societal amenities to #fffced from #fbecd7. As it was, it didn't fit well with the new tones of other colors, because it was to bright or some such. Whatever the case, the new color looks way better. So, this PR should totally be approved. Closes #1991
![before 1](https://user-images.githubusercontent.com/30259065/45940487-3e858880-bf8e-11e8-80f7-fa0302b84535.png)
![after 1](https://user-images.githubusercontent.com/30259065/45940490-41807900-bf8e-11e8-904f-6384d5045b49.png)
![before 2](https://user-images.githubusercontent.com/30259065/45940515-6117a180-bf8e-11e8-9d0e-014ec185900d.png)
![after 2](https://user-images.githubusercontent.com/30259065/45940523-712f8100-bf8e-11e8-9811-f60ba0764bf8.png)
P.S. I noticed there's an issue with the commits and files changed for some reason. I'll be fixing it shortly when I have the time.

Tomasz-W commented:
For all interested in more test renderings at different zoom levels, you can find them here: https://github.com/gravitystorm/openstreetmap-carto/issues/1991#issuecomment-423797757

kocio-pl commented:
Problems:
- gray items changeset is not needed here, it's just one-liner
- lch value in this line has changed, so please fix it

The result is much better:

https://www.openstreetmap.org/#map=15/52.2983/20.8425

Before

![grnwvwkt](https://user-images.githubusercontent.com/5439713/46048188-79bbbf00-c128-11e8-878e-ea1b5c2ec4f8.png)

After

![_ud12izq](https://user-images.githubusercontent.com/5439713/46048055-cbb01500-c127-11e8-9c9b-99956170dd42.png)

kocio-pl commented:
I would however try with more yellow, because now it's hard to recognize it from bare ground, like [here](https://www.openstreetmap.org/#map=17/52.22527/21.00267):

Before 

![8axda8q](https://user-images.githubusercontent.com/5439713/46048477-d79cd680-c129-11e8-82da-f2005feb67e8.png)


After

![ellswtlt](https://user-images.githubusercontent.com/5439713/46048437-a2908400-c129-11e8-9875-d35990529716.png)

Adamant36 commented:
@kocio-pl, thanks for the heads up. I noticed that was an issue. I keep having branch bleed over. I'll fix it when I get some time.

Tomasz-W commented:
> I would however try with more yellow

@Adamant36 Please make some test renderings with #FFFFE5

Adamant36 commented:
#FFFFE5 tests. I can do more if necessary. 
next to farmland before
![z15 next to farmland before](https://user-images.githubusercontent.com/30259065/46069959-91e40c00-c131-11e8-94c8-0e630451113c.png)
next to farmland after
![z15 next to farmland after](https://user-images.githubusercontent.com/30259065/46069996-a0322800-c131-11e8-9dfd-56fd686dfbea.png)
z14 before
![z14 before](https://user-images.githubusercontent.com/30259065/46070019-afb17100-c131-11e8-9093-5faaf3445756.png)
z14 after
![z14 after](https://user-images.githubusercontent.com/30259065/46070036-b809ac00-c131-11e8-8c91-d2ce920448a9.png)
z16 before
![z16 before](https://user-images.githubusercontent.com/30259065/46070061-c3f56e00-c131-11e8-892c-d45a45faad14.png)
z16 after
![z16 after](https://user-images.githubusercontent.com/30259065/46070086-cce63f80-c131-11e8-8559-132087e61935.png)
z17 before
![z17 before](https://user-images.githubusercontent.com/30259065/46070111-da9bc500-c131-11e8-92f7-6778fa90215d.png)
z17 after
![z17 after](https://user-images.githubusercontent.com/30259065/46070125-e091a600-c131-11e8-8264-d310e148fd0a.png)

Adamant36 commented:
@kocio-pl, I think I fixed the issues with the code. I couldn't get rid of the original bad commit. Plus, it says I added the code for bicycle repair stations and a few other things unrelated to the PR because of how I did it, but at least it doesn't have any conflicts anymore. Also, good suggestion about adding more yellow. It looks better.

kocio-pl commented:
BTW: how would the previous societal color work now? Maybe it would be enough to have new farmland color?

Tomasz-W commented:
@kocio-pl I don't understand?

kocio-pl commented:
The one we had till latest farmland change. @imagico swapped them and tuned it for new farmland, but I haven't tested if touching societal was really needed.

kocio-pl commented:
Simple revert of this color would not work at all:

![t8gftrlr](https://user-images.githubusercontent.com/5439713/46251766-033afc00-c45d-11e8-88dd-bde39a0fcc1b.png)

kocio-pl commented:
However adding a bit of yellow (I have used 5% of old parking yellow - `mix(#ffffe5, #f7efb7, 95%)` ) works OK for me against both bare ground and farmland:

![kiw6bzci](https://user-images.githubusercontent.com/5439713/46251935-020bce00-c461-11e8-94d3-b1b49993bac6.png)

![ant8byl2](https://user-images.githubusercontent.com/5439713/46251926-e30d3c00-c460-11e8-84a9-65a279b3edf1.png)
![ygnczbz3](https://user-images.githubusercontent.com/5439713/46251929-e4d6ff80-c460-11e8-98a3-5f73c925535f.png)
![e00mkzsm](https://user-images.githubusercontent.com/5439713/46251930-e6a0c300-c460-11e8-9a92-fb987d903d26.png)

kocio-pl commented:
My color is effectively #fffee3, so it looks very similar and #FFFFE5 is OK for me too, however label text should be probably darker, like 80% instead of 70%:

![u8jzza0h](https://user-images.githubusercontent.com/5439713/46252024-a8a49e80-c462-11e8-886c-709bdbf296bc.png)

70%
![4iw909vj](https://user-images.githubusercontent.com/5439713/46252063-89f2d780-c463-11e8-8f8a-116c2ffe80f3.png)

80%
![vatelih9](https://user-images.githubusercontent.com/5439713/46252058-5c0d9300-c463-11e8-9646-c084105f9da4.png)

kocio-pl commented:
#FFFFE5 with 90% darker text looks even better, I think:

![ft7axwdf](https://user-images.githubusercontent.com/5439713/46252183-b0197700-c465-11e8-85f5-a4f856edf283.png)

kocio-pl commented:
This contrast is too big for such popular landuse type like school or hospital (similar case to yellow parkings everywhere). It can work however for some less common areas - I guess culture and entertainment could be OK. Additional problem is that a lot of schools have pitch, which makes the contrast a bit painful. 

This background is clear enough for me (that's why I was trying to tune original proposition with yellow and it has really helped).

Adamant36 commented:
@turnsole80, when I test rendering the yellow color it was actually easier to spot some social amenities at higher zoom levels then it currently is. Plus, they weren't hard to distinguish from similar colored things at all, at any zoom level. So, its not necessarily the case that they will be harder to spot or anything just because the color is changed or is similar to other colors. Which it really isn't.

turnsole80 commented:
Fair enough. I can see that there is a difference between social amenities and farms, so it should be okay from my point of view.

kocio-pl commented:
@Adamant36 Could you update the code according to https://github.com/gravitystorm/openstreetmap-carto/pull/3412#issuecomment-425695508?

Adamant36 commented:
@kocio-pl, sure. I'll do it when I get a chance. Sorry for the delay.

matthijsmelissen commented:
Nice improvement, thanks @Adamant36 !

Adamant36 commented:
@kocio-pl, I updated it. Hopefully its correct now. 
![davis](https://user-images.githubusercontent.com/30259065/46603751-eb8cf480-caa8-11e8-8b76-838ef0587261.png)

Adamant36 commented on landcover.mss:
```diff
@@ -51,7 +51,7 @@
 @power-line: darken(@industrial-line, 5%);
 @sand: #f5e9c6;
-@societal_amenities: #fbecd7;   // Lch(94,12,80)
+@societal_amenities: #ffffe5;   // Lch(94,12,80)
```
@kocio-pl, my bad. I fixed it.

kocio-pl commented:
Much better now, thanks a lot!
